test/integration/gardenlet/controllerinstallation/required/required_test.go

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package care_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
+	extensionsv1alpha1 "github.com/gardener/gardener/pkg/apis/extensions/v1alpha1"
+	. "github.com/gardener/gardener/pkg/utils/test/matchers"
+)
+
+var _ = Describe("ControllerInstallation Required controller tests", func() {
+	var (
+		extensionType = "type1"
+
+		controllerRegistration *gardencorev1beta1.ControllerRegistration
+		controllerInstallation *gardencorev1beta1.ControllerInstallation
+		infrastructure         *extensionsv1alpha1.Infrastructure
+	)
+
+	BeforeEach(func() {
+		By("Create ControllerRegistration")
+		controllerRegistration = &gardencorev1beta1.ControllerRegistration{
```

Comment:
Running parallel test gives an error here - 
```
controllerregistrations.core.gardener.cloud \"ctrlreg1-kpb9d\" is forbidden: another ControllerRegistration resource already exists that controls resource Infrastructure/type1 primarily
```
Maybe it is because controller registration is a cluster-wide resource.